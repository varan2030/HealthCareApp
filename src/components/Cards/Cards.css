.card {
  background: #fff;
  border-radius: 2px;
  border-color: rgb(143, 137, 137);
  display: inline-block;
  height: 300px;
  margin: 1rem;
  position: absolute;
  width: 100%;
  box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
  transition: all 0.3s cubic-bezier(.25,.8,.25,1);
  overflow:hidden;
  padding: 15px;
  cursor: pointer;
  font-size: 15px;
  opacity: 0.8;
}

.card h3 {
  font-size: 18px;
}
a:link{
  text-decoration: none;
}

.card:hover {
  box-shadow: 0 14px 28px rgba(0,0,0,0.25), 0 10px 10px rgba(0,0,0,0.22);
}

.card-head{
  text-decoration: none;
  text-align: center;
  color: rgb(27, 27, 90);
}
