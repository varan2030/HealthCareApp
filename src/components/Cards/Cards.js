import React from 'react';
import './Cards.css';

const Cards = props => (
  <div className="card" onClick={() => props.clickCard(props.title, props.content, props.url)}>
  <div className="lang">{props.lang}</div>
    <h4 className="card-head">{props.title}</h4>
        <div dangerouslySetInnerHTML={{__html: props.content}}></div>
    </div>
)

export default Cards;