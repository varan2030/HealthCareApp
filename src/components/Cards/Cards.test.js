import React from 'react';
import Enzyme, {shallow, mount} from 'enzyme';
import Cards from "./";
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({adapter: new Adapter()});

describe("Cards component", () =>{
  
  test('renders', () => {
    const wrapper = shallow(<Cards />);
    expect(wrapper.exists()).toBe(true);
  })

  test('when the Cards is clicked the Modal is opened', () =>{
    const wrapper = (<Cards />);
    expect(wrapper).toMatchSnapshot();
  })
});
