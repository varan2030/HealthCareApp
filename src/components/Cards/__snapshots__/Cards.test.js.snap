// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards component when the Cards is clicked the Modal is opened 1`] = `<Cards />`;
